### ISSUE TITLE
Gradients are prefixed with with `-moz- old`

### ISSUE THREAD

**Issue by ivan-aksamentov** [postcss/autoprefixer#1099]
I want to prefix my CSS so that it would work on very old browsers, in particular on Firefox 5.

With `.browserslistrc` containing (solely or among other entries):
```
firefox >= 5
```

from CSS:

```
background: linear-gradient(to right, #de723f, #cebb6a, #008000, #4cbcbb, #4c4acb, #94519d);
```

autoprefixer generates `linear-gradient` (erroneously?) prefixed with `-moz- old`:

```
  background: -moz- oldlinear-gradient(left, #de723f, #cebb6a, #008000, #4cbcbb, #4c4acb, #94519d);
  background: linear-gradient(to right, #de723f, #cebb6a, #008000, #4cbcbb, #4c4acb, #94519d);
```

I am not aware of `-moz- old` prefix and Firefox does not seem to be recognizing it. Manually editing generated prefixes to `-moz-` allows Firefox 5.0.1 to display the gradient.

How can I solve this problem without manual editing?

Interestingly, [Autoprefixer Online](https://autoprefixer.github.io/), with older versions of postcss and autoprefixer, generates expected `-moz-linear-gradient`, but my repro below generates `-moz- oldlinear-gradient` (with or without downgrading packages or node)

Versions:
```
node 10.8.0
autoprefixer 9.1.2
postcss 7.0.2
```

Repro: [github.com/ivan-aksamentov/autoprefixer-repro-moz-old](https://github.com/ivan-aksamentov/autoprefixer-repro-moz-old.git)

```
git clone https://github.com/ivan-aksamentov/autoprefixer-repro-moz-old.git
cd autoprefixer-repro-moz-old
yarn install
node index.js
```

**Comment by ai** [comment]
Thanks for the perfect issue report :+1:

**Comment by ai** [comment]
Fixed 5e70d0c

**Comment by ai** [comment]
Released in 9.1.3

**Comment by ivan-aksamentov** [comment]
Whoa! Thanks for the super-quick fix.
I built from `5e70d0c` and it works for me: generates correct prefixes and FF 5 is able to render the gradient now.
